lib/matplotlib/transforms.py

```diff
@@ -807,6 +807,12 @@ def invalidate(self):
             self._check(self._points)
             super().invalidate()
 
+    def frozen(self):
+        # docstring inherited
+        frozen_bbox = Bbox(self.get_points().copy())
```

Comment:
Instead, I would call `super().frozen()` to get the base copy.

Reply:
Ah of course... It should be fixed now!